src/main/java/algorithms/easy/BestTimeToBuyAndSellStock.java

```diff
@@ -0,0 +1,24 @@
+package algorithms.easy;
+
+public class BestTimeToBuyAndSellStock {
+
+	public int maxProfit(int[] prices) {
+		int currBuy = Integer.MAX_VALUE, maxProfit = 0;
+		for (int p : prices) {
+			if (p > currBuy) {
```

Comment:
```suggestion
			if (p < currBuy) {
```
When we fix this:

![image](https://user-images.githubusercontent.com/60903744/120886166-3e6e2f80-c5f5-11eb-8ada-2d6c39b395d9.png)